vite.config.ts

```diff
@@ -8,9 +8,8 @@ export default defineConfig({
   build: {
     rollupOptions: {
       output: {
-        entryFileNames: 'DataCatalog.js',
+        entryFileNames: 'dataCatalogMain.js',
```

Comment:
Think we also need to update `.github/workflows/ci.yml` or the artifact won't upload